### PR TITLE
Add draft of release blog post

### PR DESCRIPTION
**Pull request by RedXanadu**
Add draft of release blog post covering the 4.8.0 and 3.3.7 releases.

### CODE REVIEW

**Comment by RedXanadu** [comment]
The Lychee link checker is misbehaving again.

File: content/blog/2024-10-29-crs-versions-4-8-0-and-3-3-7-released.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+author: RedXanadu
+categories:
+  - Blog
+date: '2024-10-29T18:59:58+00:00'
+title: CRS versions 4.8.0 and 3.3.7 released
+slug: 'crs-versions-4-8-0-and-3-3-7-released'
+---
+
+The OWASP ModSecurity Core Rule Set (CRS) team is pleased to announce the release of two new CRS versions: v4.8.0 and v3.3.7.
```
Comment (fzipi) [inline content/blog/2024-10-29-crs-versions-4-8-0-and-3-3-7-released.md:10]:
```suggestion
The OWASP CRS team is pleased to announce the release of two new CRS versions: v4.8.0 and v3.3.7.
```

File: content/blog/2024-10-29-crs-versions-4-8-0-and-3-3-7-released.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Other release changes
+
+Aside from the security fix, the v4.8.0 release also includes a few other minor changes and improvements as part of the normal release cycle for CRS v4. The full changes included in v4.8.0 can be found on the [GitHub release page](https://github.com/coreruleset/coreruleset/releases/tag/v4.8.0).
+
+Please feel free to contact us with any questions or concerns about this release via the usual channels: directly via the [CRS GitHub repository](https://github.com/coreruleset/coreruleset), in our Slack channel (#coreruleset on [owasp.slack.com](https://owasp.slack.com/)), or on our [mailing list](https://groups.google.com/a/owasp.org/g/modsecurity-core-rule-set-project).
+
+Sincerely,  
+Andrew Howe on behalf of the Core Rule Set development team
```
Comment (fzipi) [inline content/blog/2024-10-29-crs-versions-4-8-0-and-3-3-7-released.md:37]:
```suggestion
Andrew Howe on behalf of the CRS development team
```

**Review by fzipi — CHANGES_REQUESTED** [review]
Just two small changes.

**Comment by fzipi** [comment]
I'll be removing the check for Swiss Post after merging this one.

**Comment by RedXanadu** [comment]
Yep, yep, the changes are fine. Perhaps you can tell that I used a pre-name change release post as a template :sweat_smile:

**Pull request merged**